packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -61,21 +61,43 @@ class BuildAarCommand extends BuildSubCommand {
 
   @override
   final String description = 'Build a repository containing an AAR and a POM file.\n\n'
-      'The POM file is used to include the dependencies that the AAR was compiled against.\n\n'
+      'By default, AARs are built for `release`, `debug` and `profile`.\n'
```

Comment:
Is it possible to make it not do the default? Couldn't tell from reading the implementation. The iOS one has flags like https://github.com/flutter/flutter/blob/01f4f1ac5576a09dca32df6a2c9eaca1af1a7f22/packages/flutter_tools/lib/src/commands/build_ios_framework.dart#L41